testfile.js

```diff
@@ -0,0 +1,29 @@
+/* Generated by the Nim Compiler v1.1.1 */
```

Comment:
we shouldn't checkin generated files (unless in rare cases such as regression tests)

Reply:
I've removed it